homework_6_191122/src/Triangle.java

```diff
@@ -0,0 +1,49 @@
+// Napisat' class Triangle s parametrami storon i method vuchisleniya ploshadi i perimetra figyru.
+
+public class Triangle {
+    double sideA;
+    double sideB;
+    double baseTriangle;
+    double perimeter;
+
+    double heightTriangle;
+    double areaTriangle;
+
+    public double setSideA(double sideA) {
+        return this.sideA = sideA;
+    }
+
+    public double getSideA() {
+        return sideA;
+    }
+
+    public double setSideB(double sideB) {
+        return this.sideB = sideB;
+    }
+
+    public double getSideB() {
+        return sideB;
+    }
+
+    public double setBaseTriangle(double baseTriangle) {
+        return this.baseTriangle = baseTriangle;
+    }
+
+    public double getBaseTriangle() {
+        return baseTriangle;
+    }
+
+    public double getPerimeter() {
+        return this.perimeter = sideA + sideB + baseTriangle;
```

Comment:
refactor it

Reply:
Fixed. Changed to **return sideA + sideB + sideC;**